### PR TITLE
PT-700 Tools/artifact_node.sh Don't use "and" & "or" in the same expression

### PR DESCRIPTION
**Pull request by yvon-dblg**
Because I don't understand why my PR https://github.com/mdblp/blip/pull/164 don't build.

### CODE REVIEW

File: artifact/artifact_node.sh [hunk 1]
```diff
@@ -39,22 +47,19 @@ fi
 
 # Build Docker image whatever
 DOCKER_REPO="docker.ci.diabeloop.eu/${TRAVIS_REPO_SLUG#*/}"
-echo "Build image"
+echo "Building docker image"
 docker build --tag "${DOCKER_REPO}" --build-arg npm_token=${nexus_token} .
 
-if [ "${TRAVIS_BRANCH:-}" == "master" -a "${TRAVIS_PULL_REQUEST_BRANCH:-}" == "" -o -n "${TRAVIS_TAG:-}" ]; then
+if [ "${TRAVIS_BRANCH:-none}" == "dblp" -a "${TRAVIS_PULL_REQUEST:-false}" == "false" -a -n "${TRAVIS_TAG}" ]; then
```
Comment (franck-fourel) [inline artifact/artifact_node.sh]: if I understand correctly the build will only be possible on dblp which is not the expected behavior.

Comment (BigFab) [reply artifact/artifact_node.sh]: dont' we always have a value for TRAVIS variables?

Comment (BigFab) [reply artifact/artifact_node.sh]: _TRAVIS_BRANCH_ should never be empty
_TRAVIS_PULL_REQUEST_BRANCH_ and _TRAVIS_TAG_ can be empty

Comment (BigFab) [reply artifact/artifact_node.sh]: what behaviour do we want to achieve? publish when build push and tag? 

if [ "${TRAVIS_PULL_REQUEST:-false}" == "false" -a -n "${TRAVIS_TAG}" ]; then

**Pull request merged**